src/frontends/onnx/frontend/src/core/tensor.hpp

```diff
@@ -238,6 +239,10 @@ class Tensor {
         if (has_external_data()) {
             auto external_data = load_external_data();
             constant = std::make_shared<ngraph::op::Constant>(type, m_shape, external_data.data());
+            if (constant->get_byte_size() != external_data.size()) {
+                throw error::invalid_external_data("Size of constant " + get_name() +
```

Comment:
Please consider rewording to something like: `"The size of the external data file does not match the byte size of an initializer '" + get_name() + "' in the model".`

Reply:
It looks much better, thanks, changed